src/index.html

```diff
@@ -43,6 +43,7 @@
      <link rel="stylesheet" href="bower_components/angular-gridster/dist/angular-gridster.min.css"/>
      <script src="bower_components/javascript-detect-element-resize/jquery.resize.js"></script>
      <script src="bower_components/angular-gridster/dist/angular-gridster.min.js"></script>
+     <script src="bower_components/mousetrap/plugins/record/mousetrap-record.js"></script>
```

Comment:
This approach only works with `gulp serve` but not with `gulp build`, so it's not valid.

The three preceding lines (including angular-gridster and jquery.resize) are actually garbage and should also be removed because that code is already included in vendor.js and vendor.css.

We need a clean way to make sure mousetrap-record is included in vendor.js as well.